data/fields/barrier.json

```diff
@@ -1,5 +1,33 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
+             "bollard": "Bollard",
+             "lift_gate": "Lift Gate",
+             "retaining_wall": "Retaining Wall",
+             "block": "Large Block",
+             "guard_rail": "Guard Rail",
+             "cycle_barrier": "Cycle Barrier",
+             "stile": "Stile",
+             "entrance": "Entrance",
+             "swing_gate": "Swing Gate",
+             "cattle_grid": "Cattle Grid",
+             "ditch": "Ditch",
+             "toll_booth": "Toll Booth",
+             "city_wall": "City Wall",
+             "kissing_gate": "Kissing Gate",
+             "chain": "Chain",
+             "jersey_barrier": "Jersey Barrier",
+             "height_restrictor": "Height Restrictor",
+             "sliding_gate": "Sliding Gate",
```

Comment:
Note that barrier=sliding_gate is used regionally by paid mappers. I would not include it as a supported value.

Reply:
I will leave it out for now, though I don't really have a problem with it as the value is well documented and in widespread use...

Reply:
There is a high usage count but usage is not actually widespread.

See also https://taginfo.openstreetmap.org/tags/barrier=sliding_gate#chronology

![screen01](https://user-images.githubusercontent.com/899988/209338130-5a35973f-c953-4917-abda-569800b72e62.png)

Reply:
Distribution doesn't look too bad though, see https://taginfo.openstreetmap.org/tags/barrier=sliding_gate#map